### PR TITLE
Fix/skip benchmark documentation

### PR DESCRIPTION
**Pull request by kaushik143**
- Skip benchmark for PR with documentation label
- Trigger workflow when PR is labeled or unlabeled

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ name: Continuous Integration
 on:
   pull_request:
     branches: ['**']
-    types: [opened, synchronize, reopened, edited]
+    types: [opened, synchronize, reopened, edited, labeled, unlabeled]
```
Comment (amitksingh1490) [inline .github/workflows/ci.yml:13]: this will skip the label doc but it will rerun the Benchmarks whenever any label is added or removed

**Review by amitksingh1490 — CHANGES_REQUESTED** [review]

**Comment by tusharmath** [comment]
Should we only run benchmarks when there is a `benchmark` label attached?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kaushik-barodiya) [reply .github/workflows/ci.yml:13]: I added this so that if we remove the documentation label then again CI runs and run workflow accordingly

**Comment by amitksingh1490** [comment]
closing this due to inactivity  please reopen when you work on it again

**Pull request closed without merging**